.golangci.yml

```diff
@@ -0,0 +1,105 @@
+run:
+  skip-dirs-use-default: false
+  skip-files:
+    - ".*\\.y\\.go$"
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+  forbidigo:
+    forbid:
+      - '^fmt\.Print'
+      - '^log\.'
+      - '^print$'
+      - '^println$'
+      - '^panic$'
+  gci:
+    # Section configuration to compare against.
+    # Section names are case-insensitive and may contain parameters in ().
+    # The default order of sections is `standard > default > custom > blank > dot`,
+    # If `custom-order` is `true`, it follows the order of `sections` option.
+    # Default: ["standard", "default"]
+    sections:
+      - standard # Standard section: captures all standard packages.
+      - default # Default section: contains all imports that could not be matched to another section type.
+      - prefix(github.com/bufbuild/protocompile) # Custom section: groups all imports with the specified Prefix.
+  godox:
+    # TODO, OPT, etc. comments are fine to commit. Use FIXME comments for
+    # temporary hacks, and use godox to prevent committing them.
+    keywords: [FIXME]
+  tagliatelle:
+    # Check the struck tag name case.
+    case:
+      rules:
+        json: snake
+        yaml: snake
```

Comment:
Added this for you @mfridman.

Reply:
Appreciate it. Although may be overkill as I'm sure this will result in a lot of `//nolint:` because of the upstream JSON blobs we're interfacing with, esp. in fetcher.